src/components/RouteButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { StationContext } from './StationContextProvider';
+
+const RouteButton: React.FC<unknown> = () => {
+    const { departure, arrival } = React.useContext(StationContext);
+    const hyperlink = 'https://www.lner.co.uk/travel-information/travelling-now/live-train-times/depart/'
+    + departure.code + '/' + arrival.code + '/#LiveDepResults';

```

Comment:
It'll be nicer to use template literals here instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals